### PR TITLE
Add remodels of the Manta and its Marauder variants

### PR DESCRIPTION
**Pull request by GefullteTaubenbrust2**
## Summary
Adds remodelled sprites for the Manta and its Marauder variants.

manta ships

Manta thumbs
(Note: The game was of course modified to have all Marauders available for sale.)

## Save File
This save file can be used to view the ships in game:
[Aaron Zuhl.txt](https://github.com/endless-sky/endless-sky/files/9658443/Aaron.Zuhl.txt)

## Artwork Checklist
 - [x] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [x] I created a PR to the [endless-sky-assets repo](https://github.com/EndlessSkyCommunity/endless-sky-assets) with the necessary image, blend, and texture assets: https://github.com/EndlessSkyCommunity/endless-sky-assets/pull/85
 - [x] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: https://github.com/endless-sky/endless-sky-high-dpi/pull/306

### CODE REVIEW

**Review by Saugia — APPROVED** [review]
Hurray, no more pencil neck!  A lot of this is looking really good now! The detailing is more consistent with current ships, there are less thin parts that seem structurally dangerous, and the curved wing design was maintained which I felt was very iconic to the model.

It seems the model is much more "outlined" with details now though, especially in the back of the wings, so the design starts to lose some of its forward heaviness aspect. This is further apparent by the fact you shrunk down the size of all the panels on the wings and squeezed them between the original front lip and the new back lip you made. 

I'm not sure how I feel about it, because I do like both designs, but now I'm missing some of that front heavy aspect. It's kind of offset by how the orange ridges are more prominent on the variants, though. 

I'm sort of missing the overlapping on the rear of the engine and shield variants, where in the original thumbs you can see some of the plating pushing past the wing's rear. Overlaps can be very pleasing to the eye versus constraints like how you've done the outline around the whole wing now. However, these are more two different styles than they are something you can edit to change a bit (outlined vs overlaps), so take my opinion there as more of a personal preference than a request because both separate designs look good in their own ways.

Overall, I love it! The comments above are mostly personal nitpicks and not hard requests for redesign because the new design supports itself without having to rely on the older details. So I consider this a successful remodel. +1

File: data/human/marauders.txt [hunk 1]
```diff
@@ -632,15 +632,15 @@ ship "Marauder Manta"
...
-	gun 20 -30 "Particle Cannon"
-	turret 0 -29 "Quad Blaster Turret"
+	engine -33 40
+	engine 33 40
+	gun -20 -31 "Particle Cannon"
+	gun 20 -31 "Particle Cannon"
+	gun -26.5 -30 "Plasma Cannon"
+	gun 26.5 -30 "Plasma Cannon"
+	gun -68.5 -33 "Meteor Missile Launcher"
+	gun 68.5 -33 "Meteor Missile Launcher"
```
Comment (warp-core) [inline data/human/marauders.txt:642]: Any particular reason for moving the outermost (and now also the forwardmost) hardpoints to the end?

Comment (GefullteTaubenbrust2) [reply data/human/marauders.txt:642]: I think because I copy pasted them from the ship file. It shouldn't actually change anything though.

**Comment by GefullteTaubenbrust2** [comment]
> I'm sort of missing the overlapping on the rear of the engine and shield variants, where in the original thumbs you can see some of the plating pushing past the wing's rear.

Mh yeah I dont really agree with this one. Probably it could be done in a way that it would look good, but in the original I really feel like it is just jarring, idk.

> I'm not sure how I feel about it, because I do like both designs, but now I'm missing some of that front heavy aspect. It's kind of offset by how the orange ridges are more prominent on the variants, though.

I do get this point though. The wings I modelled are closer in profile to what you would see on a real aircraft so there isn't as nice of a curve on the backside, hence my decision to put that detail there. I'll see if I can come up with something cool though!

**Comment by Saugia** [comment]
Slight changes look good 👍

**Comment by kestrel1110** [comment]
I like it, but I will say I don't like the removal of the almost "trident-like" tail on the manta, as that was one of the coolest parts about it, and the reason that I collected those ships on all of my pilots- I do wish that could be brought back.

**Comment by Zireael07** [comment]
@kestrel1110 Seconded on the tail

**Comment by GefullteTaubenbrust2** [comment]
I kinda disliked them tbh but sure I can change it.

**Pull request merged**